<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QWidget" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTR Demo</string>
  </property>
  <layout class="QHBoxLayout" name="appWindowLayout">
   <item>
    <widget class="rtrGLWidget" name="openGLWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ui_container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="modelComboBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Duck</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trefoil</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shaderComboBox">
        <property name="currentText">
         <string>Toon</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Minimal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phong</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Toon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dots</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Procedural</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>368</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>'h' un/hide</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rtrGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>rtrglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
